templates/config-veda-vf51.json

```
@@ -0,0 +1,510 @@
+                "reg_type": "holding",
+                "address": "0x2000",
+                "type": "range",
+                "format": "u16",
+                "min": 0,
+                "max": 50,
+                "scale": 0.01,
+                "readonly": false,
+                "group": "drive_commands"
+            },
+            {
+                "name": "Frequency Lower Limit Setting",
+                "reg_type": "holding",
+                "address": "0x010D",
+                "type": "range",
+                "format": "u16",
+                "min": 0,
+                "max": 50,
+                "scale": 0.01,
+                "readonly": false,
+                "group": "drive_commands"
+            },
+            {
+                "name": "Sleep Activation",
+                "reg_type": "holding",
+                "address": "0x0D1D",
+                "format": "u16",
+                "on_value": "1",
+                "off_value": "0",
+                "type": "switch",
+                "group": "pid"
+            },
+            {
+                "name": "Set Sleep Activation Limit",
+                "reg_type": "holding",
+                "address": "0x0D1E",
+                "type": "range",
+                "format": "u16",
+                "min": 0,
+                "max": 50,
+                "scale": 0.01,
+                "readonly": false,
+                "group": "pid"
+            },
+            {
+                "name": "Running In Reverse",
+                "reg_type": "holding",
+                "address": "0x2001",
+                "format": "u16",
+                "on_value": "0x0002",
+                "off_value": "0x0000",
+                "type": "switch",
+                "group": "drive_commands"
+            },
+            {
+                "name": "Forward Launch",
+                "reg_type": "holding",
+                "address": "0x2001",
+                "format": "u16",
+                "on_value": "0x0001",
+                "off_value": "0x0000",
+                "type": "switch",
+                "group": "drive_commands"
+            },
+            {
+                "name": "Stop",
+                "reg_type": "holding",
+                "address": "0x2001",
+                "format": "u16",
+                "on_value": "0x0006",
+                "off_value": "0x0000",
+                "type": "switch",
+                "group": "drive_commands"
+            },
+            {
+                "name": "Reset Error",
+                "reg_type": "holding",
+                "address": "0x2001",
+                "format": "u16",
+                "on_value": "0x0007",
+                "off_value": "0x0000",
+                "type": "switch",
+                "group": "drive_commands"
+            }
+        ],
+        "translations": {
+            "en": {
+                "template_title": "Veda VF 51"
+            },
+            "ru": {
+                "template_title": "Veda VF 51",
```

```suggestion
            "ru": {
```
template_title нигде не используется